build.sbt

```diff
@@ -145,6 +145,15 @@ lazy val zioSchemaDerivation = crossProject(JSPlatform, JVMPlatform)
       }
     }
   )
+  .settings(
+    Test / scalacOptions ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((3, _)) =>
+          Seq("-Wconf:name=MatchCaseOnlyNullWarning:e")
+        case _ => Seq.empty
+      }
+    }
+  )
```

Comment:
Might want to remove the `scalacOptions -= "-Xfatal-warnings"` instruction that's added at a few places in the `build.sbt` file

Reply:
Ok, removed this custom setting and removed `dottySettings` from `zioSchemaDerivation` instead.